crates/holochain_state/src/query/chain_head.rs

```diff
@@ -26,8 +26,7 @@ impl Query for ChainHeadQuery {
         "
         SELECT Action.blob, Action.hash
         FROM Action
-        JOIN DhtOp ON DhtOp.action_hash = Action.hash
```

Comment:
I interpret that the JOIN on `DhtOp` is to ensure that there is an op for the potential chain head action. Unless Holochain is broken or tampered with, the actions should have been converted to ops in the authored database. If we want to test this behavior, it would be better to test it specifically.

This JOIN is what is slowing down the query. As nothing else from the DhtOp table is used in a clause, I suggest to remove it.